controllers/imageclusterinstall_controller_test.go

```diff
@@ -496,6 +522,54 @@ var _ = Describe("Reconcile", func() {
 		Expect(json.Unmarshal(content, infoOut)).To(Succeed())
 		Expect(infoOut.RawNMStateConfig).To(Equal(validNMStateConfig))
 	})
+	It("copies the nmstate config bmh preprovisioningNetworkDataName even if congimap exists", func() {
```

Comment:
Should we also have a test that we will ignore the config in the ICI if nothing is set in the BMH?

That's the behavior we're expecting, right?